frontend/src/features/admin-form/common/AdminFormLayout.tsx

```diff
@@ -47,7 +47,7 @@ export const AdminFormLayout = (): JSX.Element => {
   }
 
   return (
-    <Flex flexDir="column" height="100vh" overflow="hidden" pos="relative">
+    <Flex flexDir="column" height="100%" overflow="hidden" pos="relative">
```

Comment:
Should we set `flex=1` (and make update the parent so `flex` works)? This is so this layout component still fills the screen instead of showing white bottom padding when the content is below full viewport height. That was the original intent of setting 100vh.

Reply:
do you have a screenshot of when this happens, the layout component should still fill the screen even when content is short. if not a maybe min-height: 100vh might do the trick